GoogleForCreators/web-stories-wp
# Explore Templates: Reduce Noise for Colors in Search

## Context

We're adding in a free form search for templates on the dashboard (explore templates).

## Summary

Previous to this update, the template colors displayed as search options in explore templates have been super specific because they are based on the color label (which is used to describe the color in the template details view). We want to be more generic so users don't see options for both 'Swamp Green' and  'Dark Moss Green' - rather, just 'Green'. 

## Relevant Technical Choices

There's nothing technical here, just metadata updates to the templates package so that all colors have a `family`. Added a new test to templates to make sure there's a family added to new templates going forward, tweaked the label stored for the template search results in the dashboard package.

There were a few different names thrown out for what to call this color grouping - alias, primitive, family. I opted for family because all of the variants of orange belong to the same color family so it seemed the most straight forward. If anyone feels really strongly in a different way that's fine we can update it. 


## To-do



## User-facing changes

| New | Old | 
| --- | --- |
| ![Screen Shot 2021-11-18 at 10 33 24 AM](https://user-images.githubusercontent.com/10720454/142471910-55657774-9e84-4515-894a-701e79d91246.png) | ![Screen Shot 2021-11-18 at 11 08 38 AM](https://user-images.githubusercontent.com/10720454/142472249-e5d96b78-0825-4740-b9ef-0020194b0b60.png) |

## Testing Instructions




This PR can be tested by following these steps:

- Enable the template search experiment 
- Go to dashboard -> Explore Templates 
- Search some colors. For instance, 'Green' and see that 'Swamp Green' and 'Dark Moss Green' are no longer additional options in the list of results.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9624
